components/Table/Actions.tsx

```diff
@@ -0,0 +1,35 @@
+import Link from 'next/link'
+import { FC } from 'react'
+
+import Switch from '@/components/CustomSwitch'
+import Icons8 from '@/views/Shared/Icons8'
+
+import styles from './Actions.module.scss'
+
+interface ActionsProps {
+  editLink: string
+  showSwitch?: boolean
+  isCheckedSwitch?: boolean
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Remove this comment if not needed

Reply:
It's needed, without that eslint gives me an error that doesn't make any sense.

Reply:
Please post the screenshot of the error

Reply:
![image](https://user-images.githubusercontent.com/77207702/188475045-ea1281ec-3af9-49d4-bb54-3aefbeb71241.png)

There it is. Without the comment it gives me that warning.

Reply:
![image](https://user-images.githubusercontent.com/77207702/188475169-0fa50d4c-578f-4820-804e-76997b84eb00.png)

Reply:
Fixed, I just needed to add an underscore in value. But I think it's an error with eslint configuration. 
https://stackoverflow.com/a/61555310

Reply:
Hmm no, we left the underscore regex so devs can type interfaces like this properly

Reply:
Ah, ok.

Reply:
![image](https://user-images.githubusercontent.com/36519478/188502569-9244c3e0-ebb9-4802-a774-f2a7d51f3b35.png)
In the eslint conf you can see this rule